### PR TITLE
Implement Dark Mode

### PR DESCRIPTION
**Pull request by darkwing**
Using CSS variables will make implementing a dark mode much, much easier and more performant.

### CODE REVIEW

**Review by georgewrmarshall — COMMENTED** [review]
If my understanding is correct this implementation would add a `theme-dark` class to the root element of each page? Then we would have our theme css vars? under `:root.theme-dark`. Seems like a simple and elegant approach to me. Just wondering if this would scale to allow not only colors but other design tokens like space, font-size etc. to be able to create more themes in future?

**Comment by darkwing** [comment]
> If my understanding is correct this implementation would add a `theme-dark` class to the root element of each page? Then we would have our theme css vars? under `:root.theme-dark`. Seems like a simple and elegant approach to me. Just wondering if this would scale to allow not only colors but other design tokens like space, font-size etc. to be able to create more themes in future?

Yep!  Any CSS value can be themed using `var`!

File: ui/components/app/network-display/index.scss [hunk 1]
```diff
@@ -11,37 +11,10 @@
     cursor: not-allowed;
   }
 
-  &--colored {
```
Comment (darkwing) [inline ui/components/app/network-display/index.scss:14]: Note to self; this needs to be restored.

**Comment by Gudahtt** [comment]
@darkwing Can this be closed now that #13147 has been merged?

**Comment by darkwing** [comment]
@Gudahtt No -- this PR has the actual dark mode colors.  I've retitled the PR appropriately.

**Comment by georgewrmarshall** [comment]
@darkwing would be great to standardize these with the design system. Have booked a meeting for Thursday to discuss

**Comment by calebplum** [comment]
Any updates on this?

**Comment by jpuri** [comment]
This is great work, code and approach looks good. I think we should be good to merge incremental changes for dark mode behind feature flag.

**Comment by darkwing** [comment]
Closing as @GuillaumeRx and I have mapped out a different strategy moving forward.

**Pull request closed without merging**